src/utils/index.ts

```diff
@@ -1,6 +1,15 @@
 import Cookie from 'js-cookie';
 import router from '@/router';
 
+export function getAPIUrl(): string {
```

Comment:
Here is the important function.

Reply:
I don't see a good reason why we should store the api endpoint in a cookie. It's already set accordingly during build time in `vue.config.js`.

Reply:
It's for development/review purposes. That's why it doesn't pull from it in production.  So you can point a specific now branch deployment to a specific API now deployment in stagging.  When you make app changes that depend on API changes this allows you to test them in stagging.